### ISSUE TITLE
Help needed for AI implementations ...

### ISSUE THREAD

**Issue by darkfriend77** [HearthSim/SabberStone#24]
Anyone interested in helping in implementing some AI's

for example a Behaviour AI would be interesting ... 

also I need some other opinions in how to sync a running hearthstone game with sabberstone ...

**Comment by psiorx** [comment]
I'd be interested in helping out here but I currently can't get the project build on my Linux machine (Ubuntu 14.04).  Looks like travis is also having some issues with it.

I'll try again on a Windows box when I get a chance.

**Comment by darkfriend77** [comment]
okay, are you on Discord? Come over to the hearthsim discord channel (#stove) we might help you ...

**Comment by psiorx** [comment]
Got this to build in a 14.04 docker with the following:

```
docker run --rm -it microsoft/dotnet:1.0.5-sdk
# git clone https://github.com/HearthSim/SabberStone.git
# cd SabberStone
# dotnet restore
# dotnet sln remove SabberStoneGui/SabberStoneGui.csproj
# dotnet build
```
If there's any interest in moving some of the CI stuff into a docker to fix the build, I could set that up.

I'll join you guys in discord in a bit to discuss the current state of the AI.

**Comment by darkfriend77** [comment]
yeah would be great, if you can fix the build with docker. :-) see you on discord :-)

**Comment by Bert-Proesmans** [comment]
SabberStoneGui is a .Net project which fails building of the solution, see the Travis configuration for how to build under linux and .net core only.

`dotnet build -c Release /p:TravisBuild=True` -> this disables building the .Net projects

**Comment by douglas125** [comment]
I would like to help with the AI part. Any progress on syncing with the HS UI?

**Comment by darkfriend77** [comment]
yes ... I think we will merge some code this month which will allow ... to sync the client ... are you on discord?Am 04.02.2018 19:56 schrieb douglas125 <notifications@github.com>:I would like to help with the AI part. Any progress on syncing with the HS UI?

**Comment by chucklu** [comment]
Did you know something about this repository https://github.com/sliverfishes/silverfish?
The ai which could be used in HearthBuddy and HearthRanger.
